specification/sql/resource-manager/Microsoft.Sql/preview/2019-06-01-preview/serverOperations.json

```diff
@@ -0,0 +1,315 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2019-06-01-preview",
+    "title": "SqlManagementClient",
+    "description": "The Azure SQL Database management API provides a RESTful set of web APIs that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases, servers, and other entities."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/operations": {
```

Comment:
>/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/operations [](start = 5, length = 122)

How do you list a single operation?

Reply:
This API will list only particular operations, we currently do not support list a single operations using this api. when portal calls this API. it will return only the make all database accessible operations. but we can expand the supported operation list to return by this API.